COMETwebapp/ViewModels/Pages/UserManagement/UserManagementPageViewModel.cs

```diff
@@ -0,0 +1,165 @@
+        /// </summary>
+        public EmailAddress EmailAddress { get; set; } = new();
+
+        /// <summary>
+        ///     The <see cref="TelephoneNumber" /> to create
+        /// </summary>
+        public TelephoneNumber TelephoneNumber { get; set; } = new();
+
+        /// <summary>
+        ///     Gets or sets the data source for the grid control.
+        /// </summary>
+        public SourceList<Person> DataSource { get; } = new();
+
+        /// <summary>
+        ///    Available <see cref="Organization"/>s
+        /// </summary>
+        public IEnumerable<Organization> AvailableOrganizations { get; set; }
+
+        /// <summary>
+        ///    Available <see cref="PersonRole"/>s
+        /// </summary>
+        public IEnumerable<PersonRole> AvailablePersonRoles { get; set; }
+
+        /// <summary>
+        ///    Available <see cref="DomainOfExpertise"/>s
+        /// </summary>
+        public IEnumerable<DomainOfExpertise> AvailableDomains { get; set; }
+
+        /// <summary>
+        ///    Available <see cref="VcardEmailAddressKind"/>s
+        /// </summary>
+        public IEnumerable<VcardEmailAddressKind> EmailAddressKinds { get; set; } = Enum.GetValues(typeof(VcardEmailAddressKind)).Cast<VcardEmailAddressKind>();
+
+        /// <summary>
+        ///    Available <see cref="VcardTelephoneNumberKind"/>s
+        /// </summary>
+        public IEnumerable<VcardTelephoneNumberKind> TelephoneNumberKinds { get; set; } = Enum.GetValues(typeof(VcardTelephoneNumberKind)).Cast<VcardTelephoneNumberKind>();
+
+        /// <summary>
+        /// Injected property to get access to <see cref="ISessionAnchor"/>
+        /// </summary>
+        private readonly ISessionAnchor SessionAnchor;
+
+        /// <summary>
+        ///     Initializes a new instance of the <see cref="UserManagementPageViewModel" /> class.
+        /// </summary>
+        /// <param name="sessionAnchor">The <see cref="ISessionAnchor" /></param>
+        public UserManagementPageViewModel(ISessionAnchor sessionAnchor)
+        {
+            this.SessionAnchor = sessionAnchor;
+        }
+
+        /// <summary>
+        ///     Tries to create a new <see cref="Person" />
+        /// </summary>
+        /// <returns>A <see cref="Task" /></returns>
+        public async Task Grid_ModelSaving()
+        {
+            var thingsToCreate = new List<Person>();
+            this.Person.EmailAddress.Add(this.EmailAddress);
+            this.Person.TelephoneNumber.Add(this.TelephoneNumber);
+            thingsToCreate.Add(this.Person);
+            
+            try
+            {
+                await this.SessionAnchor.CreateThingsSiteDirectory(thingsToCreate);
+            }
+            catch (Exception exception)
+            {
+                throw;
+            }        
+  
+            this.Person = new Person();
+            this.DataSource.Clear();
+            this.DataSource.AddRange(this.SessionAnchor.GetPersons());
```

Comment:
adding the created user should happen based on a message coming from the messagebus, please have a look at COMET-IME how this is done. When use the messagebased approach then also persons added by another using in another instance of running web-app or the desktop app will appear as well on a refresh.